src/tactic/ui/panel/base_table_layout_wdg.py

```diff
@@ -1363,15 +1363,24 @@ def get_action_wdg(my):
 
         # add a custom layout widget
         custom_shelf_view = my.kwargs.get("shelf_view")
-        if not custom_shelf_view:
-            custom_shelf_view = "_layout_shelf"
+        if custom_shelf_view:
+            from tactic.ui.panel import CustomLayoutWdg
+            widget = CustomLayoutWdg(view=custom_shelf_view)
+            if widget:
+                wdg_list.append( { 'wdg': widget } )
+            else:
+                print "WARNING: shelf view '%s' not defined" % custom_shelf_view
+
+        custom_shelf_view = "_layout_shelf"
```

Comment:
the _layout_shelf (which is the old implementation) is general for all views of this search type.  shelf_view is specific to that view ... in general, it is probably better to use shelf_view ... however, if you want every table of this search type to have this, then you would use _layout_shelf.